### DISCUSSION TITLE
Assign different MiDI channels in Kit Mode

### DISCUSSION THREAD

**Discussion by Gerwingoettlich** [SynthstromAudible/DelugeFirmware#2793] [Ideas]
The idea is to assign the tracks in Kit Mode to different midi channels to play external drummers such as Syntakt. The steps should be able to set on different note values.
This would bring all drum pattern in one view.

**Comment by bukru** [comment]
it's already possible i'm sure of that because i already done that with an electribe 2.

But ... i can't remember how, but for sure it's possible : one channel per row, but only one note per row (but you can have different note on others rows).

**Reply by Gerwingoettlich** [reply]
Thanks  ! I will search for more  information.

**Reply by bukru** [reply]
while in clip view, with a KIT : press and hold audition pad of the row you want to send midi + press MIDI + turn SELECT ; )

**Reply by Gerwingoettlich** [reply]
Thank you so much ! 
i think I can delete this post for not being an idea.

**Reply by bukru** [reply]
enjoy !

**Reply by bukru** [reply]
but i don't know where i found this, i had a memory flash ^^'

**Reply by Gerwingoettlich** [reply]
I tried it like you explained. But for some reason turning select changes only the probability , even I don’t press a step it shows percentage values and trig conditions.

**Reply by bukru** [reply]
You should press thé audition pas of thé row, not a step, it is the more right button or the row

**Reply by Gerwingoettlich** [reply]
I got it . Upper golden knob for note value and the lower golden knob for midi channel.

**Reply by bukru** [reply]
And sorry i was wrong, don't turn select but top and bottom gold encoder, i tried ... Sorry

**Reply by bukru** [reply]
And i didn't sée jour message xD i should sleep !
